crates/typst/src/symbols/sym.rs

```diff
@@ -525,7 +524,10 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
     crossmark: ['✗', heavy: '✘'],
     floral: ['❦', l: '☙', r: '❧'],
     refmark: '※',
-    servicemark: '℠',
+    copyright: ['©', sound: '℗'],
+    copyleft: '🄯',
+    registered: '®',
```

Comment:
This is a symbol for a registered trademark, so I think it should be `trademark.registered`

Reply:
That makes sense